server/game/cards/03_TWI/leaders/ObiWanKenobiPatientMentor.ts

```diff
@@ -0,0 +1,43 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { LeaderUnitCard } from '../../../core/card/LeaderUnitCard';
+import { WildcardCardType } from '../../../core/Constants';
+
+export default class ObiWanKenobiPatientMentor extends LeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '2784756758',
+            internalName: 'obiwan-kenobi#patient-mentor',
+        };
+    }
+
+    protected override setupLeaderSideAbilities() {
+        this.addActionAbility({
+            title: 'Heal 1 damage from a unit',
+            cost: AbilityHelper.costs.exhaustSelf(),
+            targetResolver: {
+                cardTypeFilter: WildcardCardType.Unit,
+                immediateEffect: AbilityHelper.immediateEffects.heal({ amount: 1 })
+            }
+        });
+    }
+
+    protected override setupLeaderUnitSideAbilities() {
+        this.addOnAttackAbility({
+            title: 'Heal 1 damage from a unit',
+            targetResolver: {
+                cardTypeFilter: WildcardCardType.Unit,
+                immediateEffect: AbilityHelper.immediateEffects.heal({ amount: 1 })
+            },
+            ifYouDo: (ifYouDoContext) => ({
```

Comment:
@AMMayberry1 I have no idea how to fix ifyoudo with healsystem, can you help on this one ?